test/spec/modules/33acrossIdSystem_spec.js

```diff
@@ -157,6 +157,48 @@ describe('33acrossIdSystem', () => {
       });
     });
 
+    context('when coppa is true', () => {
+      it('should call endpoint with the coppa=1', () => {
```

Comment:
in order to make the title more readable, my suggestion is to change it to something like

`it('should call endpoint with an enabled coppa signal', () => {` 

thoughts?

Reply:
yep, will make this change